assets/js/blocks/cart/inner-blocks/cart-order-summary-taxes/index.tsx

```diff
@@ -21,7 +24,20 @@ registerFeaturePluginBlockType( metadata, {
 			/>
 		),
 	},
-	attributes,
+	attributes: {
+		...metadata.attributes,
+		...attributes,
+	},
+	title: __( 'Taxes', 'woo-gutenberg-products-block' ),
+	description: __(
+		'Shows the cart taxes row.',
+		'woo-gutenberg-products-block'
+	),
+	// description: _x(
+	// 	'Shows the cart taxes row.',
+	// 	'block description',
+	// 	'woo-gutenberg-products-block'
+	// ),
```

Comment:
All of those information should be present in the metadata file, why are you rewriting them?

Reply:
I'm overwriting the `title` & `description` to test wrapping them with the `@wordpress/i18n` functions and see if it would make any change. This worked for the `title` (it was translated correctly), but it didn't work for the `description` since the corresponding translation script/file wasn't loaded.

Reply:
Seeing other blocks using the `block.json` file (e.g. `products-by-attribute`), I think we shouldn't need the `@wordpress/i18n` functions for our translation to work! 🙌